pkg/types/json_path_expr_test.go

```diff
@@ -17,6 +17,7 @@ package types
 import (
 	"testing"
 
+	_ "github.com/go-sql-driver/mysql"
```

Comment:
Do we need import this library here?

Reply:
Yes. It asserts that the result between `MySQL` and tidb internal functions are the same. Without it, it cannot tell whether a result is correct.

Reply:
Should it added in `json_binary_test.go` file?

Reply:
You are right :facepalm: . It's a mistake. I moved the tests from `json_path_expr_test.go` to `json_binary_test.go` and forgot to modify the `import`.

It's fixed now.